bugwarrior/services/gitlab.py

```diff
@@ -299,6 +309,9 @@ def get_service_metadata(self):
         return {
             'import_labels_as_tags': self.import_labels_as_tags,
             'label_template': self.label_template,
+            'issue_priority': self.config.get('issue_priority', self.default_priority),
+            'todo_priority': self.config.get('todo_priority', self.default_priority),
+            'mr_priority': self.config.get('mr_priority', self.default_priority),
```

Comment:
This works but seems unusual. Is there any particular reason you put these in `get_service_metadata` rather than in `__init__`?

Reply:
I think, I was looking where `origin` gets filled and stumbled upon that function [here](https://github.com/ralphbean/bugwarrior/blob/develop/bugwarrior/services/__init__.py#L141). I'd be perfectly fine with moving this to the `__init__ ` method.

Reply:
Addressed in 7b026b94d445d31e3176195100eb571c1cb10ef9